src/main/java/com/sap/oss/phosphor/fosstars/data/github/VulnerabilitiesFromOwaspDependencyCheck.java

```diff
@@ -0,0 +1,415 @@
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.setOf;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sap.oss.phosphor.fosstars.data.DataProvider;
+import com.sap.oss.phosphor.fosstars.data.NoValueCache;
+import com.sap.oss.phosphor.fosstars.data.UserCallback;
+import com.sap.oss.phosphor.fosstars.data.ValueCache;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.Dependency;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspDependencyCheckEntry;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspDependencyCheckReference;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspDependencyCheckVuln;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.Software;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.VulnerableSoftware;
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import com.sap.oss.phosphor.fosstars.model.ValueSet;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.MavenArtifact;
+import com.sap.oss.phosphor.fosstars.model.value.CVSS;
+import com.sap.oss.phosphor.fosstars.model.value.Reference;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.oss.phosphor.fosstars.model.value.VersionRange;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerabilities;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability.Builder;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability.Resolution;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.owasp.dependencycheck.Engine;
+import org.owasp.dependencycheck.exception.ExceptionCollection;
+import org.owasp.dependencycheck.exception.ReportException;
+import org.owasp.dependencycheck.utils.Settings;
+
+/**
+ * This data provider tries to fill out the
+ * {@link com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures#VULNERABILITIES}. It gathers
+ * vulnerabilities about {@link MavenArtifact} using OWASP Dependency-Check.
+ * 
+ * @see <a href="https://owasp.org/www-project-dependency-check/">OWASP Dependency-Check</a>
+ */
+public class VulnerabilitiesFromOwaspDependencyCheck implements DataProvider<MavenArtifact> {
```

Comment:
If I remember correctly, you wanted to add this provider to the `experimental` package. Is that ready? Did you test it?